tasks/main.yml

```diff
@@ -23,6 +23,7 @@
     name: "{{ item }}"
     state: present
   with_items: "{{ ice_devel_packages }}"
+  when: 'ice_install_devel | default(True)'
```

Comment:
Unless there is another standard practice, my 2c is that using a dedicated YML file for storing the default variable values makes these defaults both more easier to parse and maintain.